components/tools/OmeroWeb/omeroweb/webclient/templates/webclient/annotations/metadata_general.html

```diff
@@ -105,6 +105,8 @@
                     $("#link_info_popup").show();
                 {% if webclient_path %}
```

Comment:
I kept it at first in case anyone wanted the old behaviour. If you're happy (as far as I know the Javascript below should work) I'll remove it.